flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/crd/sessionjob/FlinkSessionJob.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.kubernetes.operator.crd.sessionjob;
```

Comment:
I am thinking whether it is necessary to create a separate package `sessionjob`.

This also makes the java doc is not generated. Right?

Reply:
For this I would not use a new package just put it under spec. There are some overlapping class names with the new introduced status classes. There we could put them under `status/sessionjob/...` or rename them :)

Reply:
We could put it under the spec and status now, and make the change when necessary later and also change the doc generator.

Reply:
Fixed